src/widgets/JudgePanel.ts

```diff
@@ -571,10 +565,10 @@ export class JudgePanel extends BoxPanel {
   private _context: DocumentRegistry.IContext<JudgeModel>;
 
   private _editorWidget: CodeEditorWrapper;
-  private _markdownRenderer: IRenderMime.IRenderer;
+  private _rendermime: IRenderMimeRegistry;
   private _terminal: JudgeTerminal.IJudgeTerminal;
 
-  private _translator: ITranslator;
+  protected _translator: ITranslator;
```

Comment:
Set `translator` variable to `protected` because the inherited object (e.g. `CodleJudgeProblemPanel`) required access.